app.yaml

```diff
@@ -36,3 +36,14 @@ automatic_scaling:
 
 env_variables:
   NODE_ENV: production
+
+# Temporary workaround for a Cloud SDK bug.
+# Ensures that node_modules directory is not uploaded (the other entries are the
+# default values for skip_files).
+skip_files:
```

Comment:
This should just mirror what's in .dockerignore. No need for this to ignore py[co] files, for instance.